sources/reranker/README.md

```
@@ -0,0 +1,75 @@
+
+```yaml
+  shallow_research_agent:
+    _type: shallow_research_agent
+    llm: nemotron_nano_llm
+    tools:
+      - reranked_search
+```
+
+See `sources/reranker/example_cli_config.yml` for a full working example.
+
+### Supported Reranker Models
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)